src/lib/Libwin/accesinfo.c

```diff
@@ -80,16 +80,18 @@ create_secure_dacl(char *user, ACCESS_MASK mask, SID *owner_sid)
 	int k = 0;
 	int cbAcl = 0;
 	ACL *ndacl = NULL;
-	char logb[LOG_BUF_SIZE] = {'\0' };
+	char logb[LOG_BUF_SIZE] = {'\0' } ;
```

Comment:
why not just use log_buffer instead of defining another variable. Anyways, we are using log.h and calling logging functions from liblog

Reply:
Also, i thnk this kind of initilization of large buffers is slow. It will basically memset the entire 4096 byte with 0, whereas we probably dont need that

Reply:
since this is used for adding formatted string to logerr down somewhere, I wish there was an api "log_errf" similar to "log_eventf", so that we could have completely avoid local log bufferers

Reply:
@Shrini-h I suggest adding log_errf (see this https://github.com/PBSPro/pbspro/pull/1592#discussion_r395842996)...